OpenSim/Common/MultivariatePolynomialFunction.h

```diff
@@ -24,11 +24,11 @@
 namespace OpenSim {
 
 /** A multivariate polynomial function.
-This implementation assumes a maximum of four input dimensions and allows
+This implementation assumes a maximum of six input dimensions and allows
 computation of first-order derivatives only.
 @param coefficients the polynomial coefficients in order of ascending
-powers starting from the last dependent component.
-For a polynomial of third order dependent on three components
+powers starting from the last independent component.
+For a polynomial of third order independent on three components
```

Comment:
I think dependent was used in a different context here. Would keep it although I see how confusing it could be.

Reply:
Oh, good catch. I see why "dependent" was used now. I rephrased that comment for clarity. I'd like to keep the term "independent" everywhere else, I was getting `dimension` confused for the number of outputs.